docs/standard/serialization/binary-serialization.md

```diff
@@ -97,41 +175,162 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.Drawing.RectangleF?displayProperty=nameWithType>   
 - <xref:System.Drawing.Size?displayProperty=nameWithType>   
 - <xref:System.Drawing.SizeF?displayProperty=nameWithType>   
+- <xref:System.DuplicateWaitObjectException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.EntryPointNotFoundException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.Enum?displayProperty=nameWithType>   
 - <xref:System.Exception?displayProperty=nameWithType>   
+- <xref:System.ExecutionEngineException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.FieldAccessException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
+- <xref:System.FormatException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.Globalization.CompareInfo?displayProperty=nameWithType>   
+- <xref:System.Globalization.CultureNotFoundException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
 - <xref:System.Globalization.SortVersion?displayProperty=nameWithType>   
 - <xref:System.Guid?displayProperty=nameWithType>   
+- <xref:System.IO.Compression.ZLibException?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
The link doesn't resolve, since we haven't reflected on the type. Can you fence it and add a comment, like this:
\`System.IO.Compression.ZLibException\` \<!--zz \<xref:System.IO.Compression.ZLibException?displayProperty=fullName -->